wcfsetup/install/files/lib/system/event/listener/SessionAccessLogListener.class.php

```diff
@@ -55,7 +55,9 @@ public function execute($eventObj, $className, $eventName, array &$parameters) {
 			// Fetch request URI + request ID (if available).
 			$requestURI = UserUtil::getRequestURI();
 			if ($requestId = \wcf\getRequestId()) {
-				$requestURI .= ' ('.$requestId.')';
+				$requestIdSuffix = ' ('.$requestId.')';
+				// Ensure that the request ID fits by truncating the URI.
+				$requestURI = substr($requestURI, 0, 255 - strlen($requestIdSuffix) - 1).$requestIdSuffix;
```

Comment:
```suggestion
				$requestURI = substr($requestURI, 0, 255 - strlen($requestIdSuffix)).$requestIdSuffix;
```
Why `-1`? If the `$requestId` equals `a`, then the suffix is 4 characters long. `255 - 4 - 1 = 250`.

Reply:
I'd blame programming in C, but that would probably be a lie. You're correct of course.